src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPFormattingFeature.java

```diff
@@ -18,6 +18,19 @@
 
 /**
  * LSP formatting feature.
+ * <p>
+ * The following code snippet demonstrates how to use this class to allow a language server to override an existing
+ * formatter service:
+ * <pre>{@code
+ * public class MyLSPFormattingFeature extends LSPFormattingFeature {
+ *     @Override
+ *     protected boolean isExistingFormatterOverrideable(@NotNull PsiFile file) {
+ *         // returns true even if there is a custom formatter
+ *         return true;
+ *     }
+ * }
+ * }</pre>
+ * See the documentation of {@link #isExistingFormatterOverrideable(PsiFile)} for more details.

```

Comment:
Please add a new line to add a html link https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPApi.md#lsp-formatting-feature by formatting with an href html